ci_pod/create_ci_pod.sh

```diff
@@ -0,0 +1,60 @@
+
+## SECRETS
+
+# flask
+AWS_DEFAULT_REGION=""
+ENDPOINT_URL=""
+AWS_ACCESS_KEY_ID=""
+AWS_SECRET_ACCESS_KEY=""
+BUCKET=""
+# Add separate ip for ci pod
+POD_CI_IP=""
+BASTIAN_IP=""
+
+# Change in nginx/Dockerfile and jupyter
+JUPYTERLAB_VERSION=""
+MAINTAINER=""
+USER=""
+TOKEN=""
+
+# MUST replace MANUALLY in nginx/ibm_conf.conf
+JUPYTERLAB_URL=""
+RUN_ARTIFACTS_URL=""
+GRAFANA_URL=""
+KIBANA_URL=""
+ELASTICSEARCH_URL=""
+INTERNAL_URL=""
+
+DATA_PATH=""
+
+mkdir -p $DATA_PATH/elastic/
+mkdir -p $DATA_PATH/grafana/
+mkdir -p $DATA_PATH/windows/
+mkdir -p $DATA_PATH/jupyterLab/
+
+# chmod solved ci_pod degraded
+chmod 775 -R $DATA_PATH/elastic/
+chmod 775 -R $DATA_PATH/grafana/
+chmod 775 -R $DATA_PATH/windows/
+chmod 775 -R $DATA_PATH/jupyterLab/
+
+
+cd $DATA_PATH/ci_pod
+podman pod create --name ci_pod -p 80:80 -p "$POD_CI_IP":443:443
+podman run --name elasticsearch --pod ci_pod -d -e "discovery.type=single-node" -e "xpack.security.enabled=false" -v $DATA_PATH/elastic:/usr/share/elasticsearch/data --privileged docker.elastic.co/elasticsearch/elasticsearch:7.16.0
+podman run --name kibana --pod ci_pod -d -e "ELASTICSEARCH_HOSTS=http://localhost:9200" docker.elastic.co/kibana/kibana:7.16.0
+podman run --name grafana --pod ci_pod -d -v $DATA_PATH/grafana:/var/lib/grafana quay.io/mimehta/custom-grafana:latest
```

Comment:
Quote.  Also, this really should live elsewhere than an individual's repository.